src/plugins/newsfeed/public/components/newsfeed_header_nav_button.tsx

```diff
@@ -66,14 +66,10 @@ export const NewsfeedNavButton = ({ apiFetchResult }: Props) => {
           aria-expanded={flyoutVisible}
           aria-haspopup="true"
           aria-label="Newsfeed menu"
+          notification={showBadge ? true : null}
```

Comment:
Noice! 🎉 

You'll want to update the aria-label to indicate there there are new items. You had something similar when you built it in EUI like:

```suggestion
          aria-label={`Newsfeed menu: ${
            showBadge ? 'Updates available' : 'No updates'
          }`}
          notification={showBadge ? true : null}
```

Reply:
Oh and that all needs to he i18n'd too.